docs/en/observability/monitor-k8s/monitor-k8s-explore-logs-and-metrics.asciidoc

```diff
@@ -0,0 +1,52 @@
+[discrete]
+[[monitor-kubernetes-explore]]
+== Part 3: Explore logs and metrics
+
+Use {kib} to view the metric and log data collected by {agent}.
+Refer to the following sections for more on viewing your data.
+
+* <<monitor-k8s-explore-metrics>>.
+* <<monitor-k8s-explore-logs>>.
+
+[discrete]
+[[monitor-k8s-explore-metrics]]
+=== View performance and health metrics
+
```

Comment:
+ Ananlytics > Discover is a common page for users to search for logs/metrics and search dynamically. 
Also mentioning in case you want to include